dbus/org.sailfishos.sfmf.ufs.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Interface=org.sailfishos.sfmf.ufs
+Name=org.sailfishos.sfmf.ufs
```

Comment:
Renamed to `org.sailfishos.slipstream.upgrade` and `org.sailfishos.slipstream.unpack` now. We use "slipstreaming" in so many places, and snapshot is btrfs-specific, so not applicable to the tablet for example.